policyPrivacy.html

```diff
@@ -0,0 +1,593 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/dist/output.css" />
+    <title>Imprint Page</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Outfit:wght@100..900&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <script>
+    const loadComponents = (id, url, callback) => {
+      fetch(url)
+        .then((response) => response.text())
+        .then((html) => {
+          document.getElementById(id).outerHTML = html;
+          if (callback) callback();
+        });
+    };
+
+    window.onload = () => {
+      loadComponents("header", "src/sections/headerGreen.html", initializeMenu);
+      loadComponents("footer", "src/sections/footerGreen.html");
+    };
+
+    function initializeMenu() {
+      const menuBtn = document.getElementById("menu-btn");
+      const closeBtn = document.getElementById("close-btn");
+      const mobileMenu = document.getElementById("mobile-menu");
+
+      if (menuBtn && closeBtn && mobileMenu) {
+        menuBtn.addEventListener("click", () => {
+          mobileMenu.classList.remove("translate-x-full");
+
+          menuBtn.classList.add("hidden");
+          closeBtn.classList.remove("hidden");
+        });
+
+        closeBtn.addEventListener("click", () => {
+          mobileMenu.classList.add("translate-x-full");
+
+          menuBtn.classList.remove("hidden");
+          closeBtn.classList.add("hidden");
+        });
+
+        window.addEventListener("resize", () => {
+          if (window.innerWidth > 768 && menuBtn.classList.contains("hidden")) {
+            mobileMenu.classList.add("translate-x-full");
+            menuBtn.classList.remove("hidden");
+            closeBtn.classList.add("hidden");
+          }
+        });
+      }
+    }
+  </script>
+
+  <body class="bg-primarybg text-darkgreen">
+    <header id="header"></header>
+    <main
+      class="max-w-screen-lg mx-auto container flex flex-col space-y-8 pt-[200px] text-xl"
+    >
+      <h2 class="font-semibold">DATA PRIVACY</h2>
```

Comment:
Increase font size on smaller screens.
```suggestion
      <h2 class="font-semibold text-3xl md:text-xl">DATA PRIVACY</h2>
```